service/src/main/kotlin/fi/espoo/evaka/reports/MealReportQueries.kt

```diff
@@ -36,3 +37,15 @@ WHERE child.id = ANY (${bind(childIds)})
             )
         }
         .toMap { column<ChildId>("child_id") to row<SpecialDiet?>() }
+
+fun Database.Read.mealTexturesForChildren(childIds: Set<ChildId>): Map<ChildId, MealTexture?> =
```

Comment:
Adding `childId: null` entries to the map doesn't seem useful. Perhaps change `LEFT JOIN` to just `JOIN` and return type to `Map<ChildId, MealTexture>`.